backend/test/testUtils/testModels.ts

```diff
@@ -0,0 +1,45 @@
+import { SchemaKind } from '../../src/models/v2/Schema.js'
+
+export const testModelSchema = {
```

Comment:
This file should probably be 'testSchemas' instead of 'testModels'.

Reply:
I think there's a misunderstanding of the meaning of 'models' in this context. I'm intending this file to be the V2 version of https://github.com/gchq/Bailo/blob/main/backend/src/utils/test/testModels.ts